### ISSUE TITLE
Global value not changed in _layout.ejs after update in partial

### ISSUE THREAD

**Issue by mankewd** [sintaxi/harp#616]
I'm trying to dynamically set the page title of the pages in my static site using a recipe suggested by Raymond Camden. (https://www.raymondcamden.com/2015/07/08/using-friendlier-urls-for-a-harpjs-static-site/).  I've pared down the example to just the code I want to retrieve the pageTitle based on the file path and the code seems to be working but oddly the pageTitle global is still unset in my _layout.ejs and the rendered page does not contain the pageTitle selected from the list defined in my _data.json array. 

I've attached my code, can anyone see a problem with my structure? Or is there some timing issue in the pre-processing phase when Harp is generating the HTML files for the static site?  The HarpJS documentation states that globals are available to all elements, layout, partials, etc but it doesn't seem to work in my site. Thanks for taking a look. 

[_harp.json.txt](https://github.com/sintaxi/harp/files/1346992/_harp.json.txt)
[_layout.ejs.txt](https://github.com/sintaxi/harp/files/1346995/_layout.ejs.txt)
[_page_parser.ejs.txt](https://github.com/sintaxi/harp/files/1346993/_page_parser.ejs.txt)
[_data.json.txt](https://github.com/sintaxi/harp/files/1346994/_data.json.txt)

**Comment by sintaxi** [comment]
Is there a reason these are `.txt` files?

**Comment by mankewd** [comment]
Yes, Github doesn't allow files with ejs or json extensions to be uploaded so I just added a .txt extension to the original file names. Sorry I didn't clarify that when I logged this issue.


________________________________

**Comment by misterhtmlcss** [comment]
@sintaxi @mankewd did we get anywhere on this issue?

**Comment by sintaxi** [comment]
I'll write a test for this.
